amethyst_rendy/Cargo.toml

```diff
@@ -12,54 +12,51 @@ license = "MIT/Apache-2.0"
 features = ["shader-compiler", "test-support", "winit", "vulkan"]
 
 [dependencies]
+rendy = { git = "https://github.com/radium-io/rendy", branch = "upgrade-gfx", default-features = false, features = [
```

Comment:
need to do a rendy release when possible but can rely on branch for now if needed

Reply:
set to rev on github for now